ironfish-cli/src/utils/chainport/requests.ts

```diff
@@ -0,0 +1,64 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Could we add type annotations for the return types of these functions?

Reply:
Done!